redisinsight/ui/src/pages/rdi/pipeline-management/components/navigation/cards/jobs/JobsItem.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+
+import { Text } from 'uiSrc/components/base/text'
+import { FlexItem, Row } from 'uiSrc/components/base/layout/flex'
+import { RiTooltip } from 'uiSrc/components'
+import { DeleteIcon, EditIcon, Icon } from 'uiSrc/components/base/icons'
+import {
+  DestructiveButton,
+  IconButton,
+} from 'uiSrc/components/base/forms/buttons'
+import ConfirmationPopover from 'uiSrc/pages/rdi/components/confirmation-popover/ConfirmationPopover'
+import ValidationErrorsList from 'uiSrc/pages/rdi/pipeline-management/components/validation-errors-list/ValidationErrorsList'
+import { Indicator } from 'uiSrc/components/base/text/text.styles'
+import { ToastNotificationIcon } from '@redis-ui/icons'
+import { truncateText } from 'uiSrc/utils'
+
+type JobItemProps = {
+  name: string
+  isValid: boolean
+  validationErrors: string[]
+  isActive: boolean
+  hasChanges: boolean
+  onSelect: (name: string) => void
+  onEdit: (name: string) => void
+  onDelete: (name: string) => void
+}
+
+const JobItem = ({
+  name,
+  isValid,
+  validationErrors,
+  isActive,
+  hasChanges,
+  onSelect,
+  onEdit,
+  onDelete,
+}: JobItemProps) => (
+  <Row align="center" gap="s">
+    <FlexItem>
+      {hasChanges && (
+        <RiTooltip
+          content="This file contains undeployed changes."
+          position="top"
+        >
+          <Indicator
```

Comment:
Note: It's just a suggestion, but I propose to somehow align the jobs regardless of this indicator:

<img width="242" height="118" alt="Image" src="https://github.com/user-attachments/assets/e42b6320-aecc-4a43-bf75-04114d799878" />

 
- either align everything to the left
- or much better, align the job names to the left, no matter whether there is an indicator that highlights unsaved changes, like it's in Figma

<img width="300" height="293" alt="Image" src="https://github.com/user-attachments/assets/6934af35-1b70-4ee2-b5f1-0b7a434032a4" />